### PR TITLE
API Reference multi-pages for v1.26

### PR DESCRIPTION
**Pull request by feloy**
Update of API Reference multi-pages (https://kubernetes.io/docs/reference/kubernetes-api/) for v1.26

Preview: https://deploy-preview-38350--kubernetes-io-vnext-staging.netlify.app/docs/reference/kubernetes-api/

### CODE REVIEW

File: api-ref-assets/config/fields.yaml [hunk 1]
```diff
@@ -593,7 +596,7 @@
     - volumeName
     - storageClassName
     - volumeMode
-  - name: Beta level
+  - name: Alpha level
```
Comment (sftim) [inline api-ref-assets/config/fields.yaml:599]: This is an odd change.

**Comment by sftim** [comment]
/approve

but see https://github.com/kubernetes/website/pull/38350#pullrequestreview-1211375718

File: api-ref-assets/config/fields.yaml (hunk 1 above)
Comment (feloy) [reply api-ref-assets/config/fields.yaml:599]: I agree, but this is reflecting the change of the comment of the field

Comment (feloy) [reply api-ref-assets/config/fields.yaml]: I have pushed a commit to change Alpha -> Beta in the fields.yaml file.

Note that the generated HTML content is OK, as it was already containing Beta.

**Comment by krol3** [comment]
cc: @onlydole

**Comment by nate-double-u** [comment]
As a part of our release process, we've merged 1.26 into main, so this will now need to target `main` as well.

**Comment by nate-double-u** [comment]
closing in favour of https://github.com/kubernetes/website/pull/38364

/close

**Pull request closed without merging**